minutes/20200222.md

```diff
@@ -0,0 +1,37 @@
+# CILG 9
+
+## Attendess
+- mauve
+- benedict
+- yurko
+- austin
+- greg
+
+## minutes
+- yurko: working on cross compiling nodejs
+- ben: if i'm on x64 i can install this package
+- yurko: anything but ARM right now
+- mauve: exokit specifies a CLI flag for npm i.
+- yurko: part of ssb uses native binaries.
+- ben: how does install work?
+- yurko: dpkg i
+- ben: how does package hosting work
+- yurko: nginx
+- ben: how do you admin the pub through a web page
+- austin: ssb-client allows rpc commands over websocket
+- ben: we could use vpn to restrict access via IP address
+- austin:
+- yurko: travis does compliation and zips it all up
+- ben: deb packages can also ship source, not just binaries
+- mauve: where can i help, interested in testing deb install on multiple platforms, can possibly help with cross-compiling for arm
+- yurko: not sure where to start with compiling for ARM
+
```

Comment:
```suggestion
- ben: we trialed _aether pro_-based email group as main coordination since last meeting and agreed to go ahead with it
  - hope it will improve async workflow moving forward
  - re-introduced how to use the email group and shared self-registration link
  - yurko sent an email to group address, creating a new thread for dev + test of the debian packages, and others received it
```